crates/duckdb/src/appender/mod.rs

```diff
@@ -287,22 +289,23 @@ mod test {
     }
 
     #[test]
-    fn test_appender_error() -> Result<(), crate::Error> {
+    fn test_appender_computed_columns() -> Result<(), crate::Error> {
         let conn = Connection::open_in_memory()?;
         conn.execute(
             r"CREATE TABLE foo (
             foobar TEXT,
-            foobar_split TEXT[] AS (split(trim(foobar), ','))
+            foobar_twice TEXT AS (foobar || foobar)
```

Comment:
Scalar type makes testing using `query_row` a bit easier (i couldn't figure out how to use query_row for a `(String, Vec<String>)`, got some odd type errors.